### PR TITLE
Show events in web-interface with help of toasts

### PR DESCRIPTION
**Pull request by Abyss777**
I think I've managed to find a way how send a complete data (an event and a position) to show in the  web-interface.

### CODE REVIEW

File: web/app/store/Events.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Ext.define('Traccar.store.Events', {
+    extend: 'Ext.data.Store',
+    model: 'Traccar.model.Event',
+
+    proxy: {
+        type: 'rest',
+        url: '/api/events'
+    }
+});
```
Comment (tananaev) [inline web/app/store/Events.js]: New line is missing. By the way, there is style and issues check for JavaScript code as well. Use `jshint` and `jscs`.

File: web/app/controller/Root.js [hunk 2]
```diff
@@ -126,6 +126,35 @@ Ext.define('Traccar.controller.Root', {
                     }
                 }
             }
+
+            if (data.events) {
+                array = data.events;
+                store = Ext.getStore('Events');
+                for (i = 0; i < array.length; i++) {
+                    store.add(array[i]);
+                    var text;
```
Comment (tananaev) [inline web/app/controller/Root.js]: All variables should be defined at the beginning of the method.

File: src/org/traccar/api/AsyncSocket.java [hunk 3]
```diff
@@ -45,7 +47,9 @@ public AsyncSocket(long userId) {
     public void onWebSocketConnect(Session session) {
         super.onWebSocketConnect(session);
 
-        sendData(KEY_POSITIONS, Context.getConnectionManager().getInitialState(userId));
+        Map<String, Collection<?>> data = new HashMap<String, Collection<?>>();
```
Comment (tananaev) [inline src/org/traccar/api/AsyncSocket.java]: Traccar targets Java 7 and higher, so you can use diamond operator on the right hand side.

**Comment by tananaev** [comment]
Added some comments.

**Comment by Abyss777** [comment]
Fixed. Thanks for the tip about JSHint.

**Comment by tananaev** [comment]
Merged. Thanks.

**Pull request merged**